packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+    EuiAvatar,
+    EuiComment,
+    EuiCommentList,
+    EuiSpacer,
+    EuiText,
+} from '@elastic/eui';
+import { useTranslations } from 'next-intl';
+
+import { useWithOrchestratorTheme } from '../../hooks';
+import { WFOProcessStatusBadge } from '../WFOBadges';
+import { SubscriptionDetailProcess } from '../../types';
+import { WFOLoading } from '../WFOLoading';
+import { PATH_PROCESSES } from '../WFOPageTemplate';
+import { parseDateToLocaleDateTimeString, parseDate } from '../../utils';
+import { upperCaseFirstChar } from '../../utils';
+
+import { getStyles } from './styles';
+
+interface WfoProcessCardProps {
+    subscriptionDetailProcess: SubscriptionDetailProcess;
+}
+
+const WfoProcessCard = ({ subscriptionDetailProcess }: WfoProcessCardProps) => {
+    const t = useTranslations('subscriptions.detail.processDetail');
+    const {
+        tableStyle,
+        contentCellStyle,
+        headerCellStyle,
+        emptyCellStyle,
+        lastContentCellStyle,
+        lastHeaderCellStyle,
+    } = useWithOrchestratorTheme(getStyles);
+
+    return (
+        <div style={{ marginTop: 5 }}>
```

Comment:
Css?

Reply:
Indeed, css

Reply:
I can move the css but I worked with the assumption that we still do oneliners like this inline?

Reply:
Move the style to the table style and removed this one